### PR TITLE
add red hat font to 3scale

### PR DESCRIPTION
**Pull request by christiemolloy**
**What this PR does / why we need it**:

> Add the new red hat font to the UI

**Which issue(s) this PR fixes** 

> Issue on Jira: https://issues.redhat.com/browse/IUXD-46

**Verification steps**

> Check all pages to make sure that font has been applied to all features. 

**Special notes for your reviewer**:

Screen Shot 2019-12-17 at 11 43 33 AM

Screen Shot 2019-12-17 at 11 43 33 AM

Screen Shot 2019-12-17 at 11 44 08 AM

### CODE REVIEW

**Comment by josemigallas** [comment]
@christiemolloy 
Is it necessary to upgrade patternfly-react for this change? If not, could you please remove it from this PR?

**Comment by josemigallas** [comment]
Also, I guess this changes makes the rest of fonts useless, therefore their removal belongs to this PR. Or do they? @thomasmaas @damianpm

**Comment by damianpm** [comment]
@christiemolloy I've changed the target branch to `redhat-font`, because I'd like to deploy to our preview environment and test it before merging to master

**Comment by damianpm** [comment]
> @christiemolloy
> Is it necessary to upgrade patternfly-react for this change? If not, could you please remove it from this PR?

@christiemolloy  If not necessary to update the packages, please remove it, we can do that in another PR.

Thank you so much!

**Comment by damianpm** [comment]
> Also, I guess this changes makes the rest of fonts useless, therefore their removal belongs to this PR. Or do they? @thomasmaas @damianpm

Maybe we can't remove the other fonts, they may be in use in the developer portal.
I'd let that for a subsequent PR

**Comment by thomasmaas** [comment]
Probably fonts can be removed. Needs to be checked. If so it does belong in this pr.

**Comment by christiemolloy** [comment]
Removed change that upgraded the packages @josemigallas @damianpm . Let me know if you want the old fonts removed too! @thomasmaas

**Comment by christiemolloy** [comment]
@damianpm let me know if theres anything else to add

File: app/views/layouts/provider/login.html.slim [hunk 1]
```diff
@@ -1,12 +1,11 @@
 doctype html
-html lang="en"
+html lang="en" class="pf-m-redhat-font"
```
Comment (damianpm) [inline app/views/layouts/provider/login.html.slim]: Just to keep consistency, can we use the same format like the other 2 layouts?
`html[lang="en" class="pf-m-redhat-font"]`

Comment (christiemolloy) [reply app/views/layouts/provider/login.html.slim]: good point, updated!

**Comment by damianpm** [comment]
Thanks a lot @christiemolloy !!
Now I'm going to deploy to our preview environment and run some tests
It should be ready soon
🎉

**Pull request merged**